src/test/java/DownloadTest.java

```diff
@@ -0,0 +1,47 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+import static constants.LocatorsConstants.*;
+import static constants.MainConstants.LINK_SELENIUM_DOWNLOAD;
+import static org.testng.Assert.assertEquals;
+
+public class DownloadTest {
+    private WebDriver driver;
+
+    @BeforeClass
+    void setup () {
```

Comment:
could this method be moved to base test, which will be parent for all of your tests?